app/models/budget/investment.rb

```diff
@@ -233,6 +233,7 @@ def reason_for_not_being_ballotable_by(user, ballot)
       return :not_selected                    unless selected?
       return :no_ballots_allowed              unless budget.balloting?
       return :different_heading_assigned_html unless ballot.valid_heading?(heading)
+      return :not_enough_available_votes_html if heading.group.approval_voting? && ballot.investments.count == heading.group.number_votes_per_heading
```

Comment:
How about moving these conditionals to a method?

Reply:
Could you review the other @houndci-bot alerts? 🙏

Reply:
Yes of course. I remember doing some thing like that, may be another project.